forms.py

```diff
@@ -10,7 +10,30 @@
 from wtforms.fields.html5 import EmailField
 
 
-def strip_filter(value):
+# amount must be $1 or higher
+def validate_amount(form, field):
+    value = field.data
+    if value is None:
+        raise validators.ValidationError("Invalid amount")
+    if value < 1:
+        raise validators.ValidationError("Amount is less than 1")
+
+
+# if value starts with a dollar sign, remove it
+# then convert to a float
+def format_amount(value):
+    if value.startswith("$"):
+        value_to_convert = value[1:]
+    else:
+        value_to_convert = value
+    try:
+        float(value_to_convert)
+        return float(value_to_convert)
```

Comment:
Not a big deal but you could replace these with:
```
       value = value[1:]
try:
   return float(value)
```
The return is in the `try` block so you don't need to do the cast twice.